apps/src/templates/projects/PersonalProjectsNameCell.jsx

```diff
@@ -29,7 +29,14 @@ class PersonalProjectsNameCell extends Component {
     return (
       <div>
         {!isEditing &&
-          <a style={tableLayoutStyles.link} href={url} target="_blank">{projectName}</a>
+          <a
+            style={tableLayoutStyles.link}
+            href={url}
+            target="_blank"
+            className={`${projectName}`}
```

Comment:
I think this is identical to `className={projectName}`?

Reply:
Uh, yep, sure is.  I was considering appending "ui-" to the front to be consistent with other classNames I'm using only for UI tests, but then abandoned that because I didn't like the wording in my step definition. Whoops! Thanks for catching.